internal/runners/deploy/deploy.go

```diff
@@ -173,7 +173,7 @@ func (d *Deploy) install(rtTarget setup.Targeter) error {
 	// Todo Remove with https://www.pivotaltracker.com/story/show/178161240
 	// call rti.Environ as this completes the runtime activation cycle:
 	// It ensures that the analytics event for failure / success are sent
-	_, _ = rti.Env(false, false)
+	_, _ = rti.Env(false, false, false)
```

Comment:
There are a lot of cases here where we are defaulting to `false` for `bashifyPath` is there a reason why we don't want to bashify these paths?

Reply:
When we have a lot of arguments that are just booleans it may be worth considering comments to explain them. Though I realize that's a lot of comment writing in this case so I'll leave it up to you.

Reply:
That's fair. Again, comments might be helpful.